jpsreport/CMakeLists.txt

```diff
@@ -319,11 +319,13 @@ target_link_libraries(report PUBLIC
     tinyxml
     fs
     Boost::boost
+    git-info
 )
 
 target_compile_definitions(report PUBLIC
     PYTHON="${PYTHON_EXECUTABLE}"
     PYTHON_VERSION="${PYTHON_VERSION_STRING}"
+    JPSREPORT_VERSION="0.8.5"
```

Comment:
Why do you set the version here manually?

Reply:
Ah right, I wanted to ask you or @zeroset if both report and core should use the same version number but then I forgot. So should they share the version number?

Reply:
A good question for @zeroset and @chraibi.

Reply:
I talked to @zeroset in chat and the conclusion was that core and report will use the same verison number from now on

Reply:
ok